multiapi-engine/src/test/resources/asyncapigenerator/testCustomValidators/assets/DataDTO.java

```diff
@@ -0,0 +1,202 @@
+
+    public DataDTO.DataDTOBuilder clientId(Integer clientId) {
+      this.clientId = clientId;
+      return this;
+    }
+
+    public DataDTO.DataDTOBuilder clientName(String clientName) {
+      this.clientName = clientName;
+      return this;
+    }
+
+    public DataDTO.DataDTOBuilder flightNumber(String flightNumber) {
+      this.flightNumber = flightNumber;
+      return this;
+    }
+    public DataDTO.DataDTOBuilder test(List<integer> test) {
+      if (!test.isEmpty()) {
+        this.test.addAll(test);
+      }
+      return this;
+    }
+
+    public DataDTO.DataDTOBuilder tes(integer tes) {
+      if (tes != null) {
+        this.test.add(tes);
+      }
+      return this;
+    }
+
+    public DataDTO build() {
+      DataDTO dataDTO = new DataDTO(this);
+      return dataDTO;
+    }
+  }
+
+  /**
+  * Get clientId
+  * @return clientId
+  */
+  @Schema(name = "clientId", required = true)
+  public Integer getClientId() {
+    return clientId;
+  }
+
+  /**
+  * Get clientName
+  * @return clientName
+  */
+  @Schema(name = "clientName", required = true)
+  public String getClientName() {
+    return clientName;
+  }
+
+  /**
+  * Get flightNumber
+  * @return flightNumber
+  */
+  @Schema(name = "flightNumber", required = true)
+  public String getFlightNumber() {
+    return flightNumber;
+  }
+
+  /**
+  * Get test
+  * @return test
+  */
+  @Schema(name = "test", required = true)
+  public List<integer> getTest() {
+    return test;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    DataDTO dataDTO = (DataDTO) o;
+    return Objects.equals(this.clientId, dataDTO.clientId) && Objects.equals(this.clientName, dataDTO.clientName) && Objects.equals(this.flightNumber, dataDTO.flightNumber) && Objects.equals(this.test, dataDTO.test);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(clientId, clientName, flightNumber, test);
+  }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("class DataDTO {\n");
+    sb.append(" clientId: ").append(toIndentedString(clientId)).append("\n");
+    sb.append(" clientName: ").append(toIndentedString(clientName)).append("\n");
+    sb.append(" flightNumber: ").append(toIndentedString(flightNumber)).append("\n");
+    sb.append(" test: ").append(toIndentedString(test)).append("\n");
+    sb.append("}");
+    return sb.toString();
+  }
+
+  /**
+  * Convert the given object to string with each line indented by 4 spaces
+  * (except the first line).
+  */
+  private String toIndentedString(Object o) {
+    if (o == null) {
+      return "null";
+    }
+    return o.toString().replace("\n", "\n ");
+  }
+
+
+  private void validateRequiredAttributes() {
+    boolean satisfiedCondition = true;
+
+    if (!Objects.nonNull(this.clientId)) {
```

Comment:
Why not?
```
 if (!Objects.nonNull(this.clientId) || !Objects.nonNull(this.clientName) || !Objects.nonNull(this.flightNumber) || !Objects.nonNull(this.test)) {
  satisfiedCondition = false;
}
```

Reply:
It is a good idea to change this code, but it is automatically generated and should be done in another issue.